L1Trigger/TrackerDTC/test/test_cfg.py

```diff
@@ -9,14 +9,16 @@
 
 process = cms.Process( "Demo" )
 process.load( 'FWCore.MessageService.MessageLogger_cfi' )
-process.load( 'Configuration.Geometry.GeometryExtended2026D76Reco_cff' ) 
-process.load( 'Configuration.Geometry.GeometryExtended2026D76_cff' )
+process.load( 'Configuration.Geometry.GeometryExtended2026D98Reco_cff' ) 
```

Comment:
By the way, this will not work. We already change 2026 to Run4. See https://github.com/cms-sw/cmssw/pull/46582